fastapi_cache/decorator.py

```diff
@@ -2,6 +2,7 @@
 import sys
 from functools import wraps
 from typing import Any, Awaitable, Callable, Optional, Type, TypeVar
+import logging
```

Comment:
You may need to refer to the coding style of PEP8, the `import` statement should be above the`from`, like this:


```suggestion
import logging
import sys
from functools import wraps
from typing import Any, Awaitable, Callable, Optional, Type, TypeVar
```

@long2ice This also reminds us that we need to update the CI to achieve the purpose of the unified code style.